src/components/com_tjcertificate/administrator/languages/en-GB/en-GB.com_tjcertificate.ini

```diff
@@ -383,3 +383,24 @@ COM_TJCERTIFICATE_PERMISSION_DELETE_EXTERNAL_CERTIFICATE="Delete training record
 COM_TJCERTIFICATE_PERMISSION_DELETE_EXTERNAL_CERTIFICATE_DESC="Allow users to delete training records"
 COM_TJCERTIFICATE_PERMISSION_DELETEOWN_EXTERNAL_CERTIFICATE="Delete own records"
 COM_TJCERTIFICATE_PERMISSION_DELETEOWN_EXTERNAL_CERTIFICATE_DESC="Allow users to delete own training records"
+
+;Bulk Record Config
+COM_TJCERTIFICATE_TJQUEUE_SETTING="Use TjQueue while add bulk records"
+COM_TJCERTIFICATE_TJQUEUE_SETTING_DESC="Set to 'Yes' if you want to use TjQueue while add bulk records (Note: To work this functionality you need com_tjqueue and certificate plugin enable)"
+COM_TJCERTIFICATE_RECORDS_USER_SELECT_LIMIT="User limit for add bulk records"
+COM_TJCERTIFICATE_RECORDS_USER_SELECT_LIMIT_DESC="This limit used only when you are not using TjQueue and limit above 40 will result timeout."
```

Comment:
This limit value is used only when you are not using TjQueue and if the limit is set above 40 may result in a timeout.

Reply:
We are using limit with or without tjqueue so updating text as below